guides/common/modules/con_disaster-recovery-by-virtualizing-your-projectserver.adoc

```diff
@@ -0,0 +1,10 @@
+[id="disaster-recovery-by-virtualizing-your-{project-context}-server"]
+= Disaster recovery by virtualizing your {ProjectServer}
+
+If you virtualize your {ProjectServer} and ensure that you take regular snapshots of the virtual machine (VM), you can respond to various disaster scenarios by restoring your {Project} deployment from one of your snapshots.
+
+[NOTE]
+====
+The details for how to implement this scenario depend on your choice of a virtualization platform.
+Due to the variety of different hypervisors and their capabilities, {Team} does not provide detailed instructions for any specific hypervisor.
```

Comment:
```suggestion
Due to the variety of different hypervisors and their capabilities, {Team} does not provide detailed instructions for any specific virtualization platform.
```

It might be me, but when you say "hypervisor", I think of the physical entity of hardware and software that runs my vm (so an RHV host, an VMware ESXi host, etc), but the snapshotting/restoring/moving is a feature of the overall platform (so, RHV, VMware vCenter, etc)

Feel free to ignore ;)